tests/sav/nitsmells_args1.res

```diff
@@ -0,0 +1,18 @@
+Test ModelBuilder
+*** CODE SMELLS METRICS ***
+--- Code Smells Metrics ---
+-------------------
+Class: Character
+Feature envy :  true
+	Affected method:
+		-quit
+Long method:  Average 1 lines
+	Affected method:
+		-total_strengh has 2 lines
+		-total_endurance has 2 lines
+		-total_intelligence has 2 lines
+-------------------
+Class: Character
```

Comment:
If it appears twice for the same program, you are not talking about `Class` but `Classdef`